about.html

```diff
@@ -0,0 +1,114 @@
+    <nav class="mobile_nav">
+      <button class="hamburger">menu
+        <i class="fa-sharp fa-solid fa-xmark close whiteColor">
+        </i>
+        <i class="fa-solid fa-bars open"></i>
+      </button>
+      <a href="./index.html" class="lelogo"><img src="./pictures/logoLion.png" class="logoLion" alt="Museum_logo" srcset=""></a>
+      <ul class="menu_list">
+        <li><a class="link_item whiteColor" href="./index.html">Home</a></li>
+        <li><a class="link_item whiteColor" href="">Program</a></li>
+        <li><a class="link_item whiteColor" href="">Join</a></li>
+        <li><a class="link_item whiteColor" href="">Sponsor</a></li>
+        <li><a class="link_item whiteColor" href="">News</a></li>
+        <li><a class=" compaign" href="">NC Compaign</a></li>
+      </ul>
+    </nav>
+    <div class="banner">
+      <q class="salute">Hello! sharing world</q>
+      <h1 class="bigheader">NIGER CONNEXION<br> GLOBAL SUMMIT 2015</h1>
+      <div class="paragraph whiteBackground">
+        <p>
+NIGER CONNEXION is an event agency whose niche is cultural entrepreneurship. Founded in 2009 and holder of a license
+issued by the APEIC (Agence de Promotion des Entreprises et Industries Culturelles), composed of a young and dynamic
+team always in search of performance and innovation.
+
+Associated with all the major cultural and artistic events in Niger since its creation, it is also equipped with modern
+equipment to match its performance.
+        </p>
+      </div>
+      <div class="timeZone">
+        <p class="message">Please Contact us per Email for any further questions about CC Global Summit 2015</p>
+        <p class="place">ncsummit2015@cckorea.org</p>
+      </div>
+    </div>
+  </header>
+  <main class="container">
+    <div class="programs whiteBackground">
+      <h2 class="title">Niger Connexion  Global Summit 2015 Logo</h2>
+      <hr class="red_bar">
+      <p class="message">The logo of NC Global Summit 2015 was decided <br> through the logo competition from 8. June to 7 . July</p>
+        <div class="card_program">
+          <img src="./pictures/logoLion.png" alt="logo" class="centerLogo">
+        </div>
+
+    </div>
+    <div class="speakers">
+      <h2 class="title">See the past NC Global Summit</h2>
+      <hr class="red_bar2">
+      <p class="message">Take a look at the last two NC Global Summit which took place in Niamey and in Maradi.</p>
+      <div class="imagesFlex whiteColor">
+              <div class="overlay">
+                <div class="aboutCard1">
+                  <h2>2013</h2>
+                  <h5>NC Global SUmmit 2013 in Niamey</h5>
+                </div>
+              </div>
+              <div class="overlay">
+                <div class="aboutCard2">
+                  <h2>2011</h2>
+                  <h5>NC Global SUmmit 2011 in Maradi</h5>
+                </div>
+              </div>
+      </div>
+
+    </div>
+    <div class="partners whiteColor">
+      <h3>Partner</h3>
```

Comment:
- Kindly, update the partner header from `h3` to `h2` as the previous code reviewer suggested you for better sematic reasons 👍